plonky2/src/fri/reduction_strategies.rs

```diff
@@ -8,11 +8,12 @@ pub enum FriReductionStrategy {
     /// Specifies the exact sequence of arities (expressed in bits) to use.
     Fixed(Vec<usize>),
 
-    /// `ConstantArityBits(arity_bits, final_poly_bits)` applies reductions of arity `2^arity_bits`
-    /// until the polynomial degree is `2^final_poly_bits` or less. This tends to work well in the
-    /// recursive setting, as it avoids needing multiple configurations of gates used in FRI
-    /// verification, such as `InterpolationGate`.
-    ConstantArityBits(usize, usize),
+    /// `ConstantArityBits(arity_bits, final_poly_bits, cap_height)` applies reductions of arity `2^arity_bits`
+    /// until the polynomial degree is less than or equal to `2^final_poly_bits` or until any further
+    /// `arity_bits`-reduction makes the last FRI tree have height less than `cap_height`.
+    /// This tends to work well in the recursive setting, as it avoids needing multiple configurations
+    /// of gates used in FRI verification, such as `InterpolationGate`.
+    ConstantArityBits(usize, usize, usize),
```

Comment:
It seems like there would be a bit of redundancy in the config if we add `cap_height` here; could we make it a param to `reduction_arity_bits` that comes from the existing config value?

Reply:
Oh yeah good idea, done.